.ci.yaml

```diff
@@ -2408,6 +2408,16 @@ targets:
         ["devicelab", "android", "linux", "pixel", "7pro"]
       task_name: flavors_test
 
+  - name: Linux asset_transformation
+    presubmit: false
+    bringup: true
+    recipe: flutter/flutter_drone
+    timeout: 60
+    properties:
+      tags: >
+        ["hostonly","devicelab","linux"]
```

Comment:
Is there any reason this should be a devicelab test?  It looks like it should be a tools integration test and not take up the slot of a Linux machine with a tethered Android device https://github.com/flutter/flutter/tree/master/packages/flutter_tools/test/integration.shard

Reply:
Good catch

Reply:
I've rewritten this to be a test that runs on the tool integration test shard as requested. Thanks for the suggestion! I also didn't consider that devicelab tests always occupy a target device, even if they aren't used by the test (assuming I understand correctly).